components/dialog/stories/dialog.stories.js

```diff
@@ -46,39 +116,186 @@ export default {
 			},
 			control: "boolean",
 		},
+		heroImageUrl: {
```

Comment:
It'd be nice if I could view the hero image from the controls, without having to upload an image. Are we missing the `hasHeroImage` control? I see it referenced once in the Docs story.

Reply:
Let me know if this is close to what you were thinking! https://github.com/adobe/spectrum-css/pull/2860/commits/9d8f8813c30cfd39bca1ad678b3345986ed54687